.github/workflows/adhoc.yml

```
@@ -83,12 +87,13 @@
           username: guarzo
           password: ${{ secrets.DOCKERHUB_PAT }}
 
-      - name: Build and Push Docker image
+      - name: Build Docker image (SHA tagged)
         uses: docker/build-push-action@v4
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)